annotation/src/main/resources/application.properties.EXAMPLE

```diff
@@ -1,5 +1,6 @@
 vep.url=http://grch37.rest.ensembl.org/vep/human/hgvs/VARIANT?content-type=application/json&xref_refseq=1&ccds=1&canonical=1&domains=1&hgvs=1&numbers=1&protein=1
 hotspots.url=http://cancerhotspots.org/api/hotspots/single/
-vep.overrides_uris=uniprot=isoform_overrides_uniprot.txt,mskcc=isoform_overrides_at_mskcc.txt
+vep.overrides_uris=uniprot:isoform_overrides_uniprot.txt,mskcc:isoform_overrides_at_mskcc.txt
 
-spring.data.mongodb.uri=
+spring.data.mongodb.uri=mongodb://127.0.0.1:27017/annotator
```

Comment:
I think leaving this line blank is better, or replacing the address with `<HOST>`, `<PORT>`, and the db name to `<DB>`. Same thing for the `server.port` value.